### DISCUSSION TITLE
How can I show custom popup menu in q-select component?

### DISCUSSION THREAD

**Discussion by FragsterAt** [quasarframework/quasar#16438] [General - Components / Directives / etc]
I want to make beautiful scrollbars in the drop-down menu, like here https://codepen.io/FragsterAt/pen/wvRRjZm?editors=1010 , but I didn't find either a slot or an event to cancel the display of the built-in menu and show my own instead. Is there any way to do what I want?

**Comment by metalsadman** [comment]
maybe you don't need the q-select then? you can use option slot, but that's just for the items.

**Reply by FragsterAt** [reply]
I need a lot of q-select features, but I also need beautiful scroll bars on the popup menu

**Reply by metalsadman** [reply]
try with `::-webkit-scrollbar` and `::-webkit-scrollbar-thumb` css.
